tests/app/settings.py

```diff
@@ -118,7 +118,10 @@
         'CACHE': False,
         'BUNDLE_DIR_NAME': 'bundles/',
         'STATS_FILE': os.path.join(BASE_DIR, 'webpack-stats-app2.json'),
-    }
+    },
+    'NO_IGNORE': {
```

Comment:
`NO_IGNORE_APP` is probably a better name, the NO_IGNORE -> IGNORE bugged my mind for few seconds 😂

Reply:
Done